playbooks/roles/suse-build-images/tasks/build-with-loci.yml

```diff
@@ -6,7 +6,7 @@
 - name: Build loci images in parallel (using OSH-images tooling)
   shell: "{{ loci_build_command | default('./openstack/loci/build.sh') }}"
   args:
-    chdir: "{{ upstream_repos_clone_folder }}/openstack-helm-images"
+    chdir: "{{ upstream_repos_clone_folder }}/openstack/openstack-helm-images"
   environment: "{{ build_environment | default({}) | combine(default_environment) }}"
   vars:
     default_environment:
```

Comment:
Next line 13 needs to be updated as well (not shown in review here).

LOCI_SRC_DIR: "{{ upstream_repos_clone_folder }}/loci"  (need to have /openstack/loci )